### PR TITLE
Maintain basic scenarios

### PR DESCRIPTION
**Pull request by IKukhta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +37,19 @@ To describe types and instances that are associated with the CTI identifiers, RA
 go get -u github.com/acronis/go-cti
 ```
 
-### CLI
+## CLI
```
Comment (yury-palyanitsa) [inline README.md:40]: This is a part of Installation section so you should keep h3 here.

File: README.md [hunk 2]
```diff
@@ -79,12 +79,33 @@ Example:
 cti validate
 ```
 
-##### cti pack
+### cti pack
 
-Packs the package into a bundle.
+Packs the package into a bundle. Valid package should be in current working directory (or directory specified by `--working-dir`).
```
Comment (yury-palyanitsa) [inline README.md:84]: Current working directory and global flag is already mentioned in the note above. I think it's not necessary here.

File: README.md [hunk 3]
```diff
@@ -79,12 +79,33 @@ Example:
...
+Includes the source files into the bundle. By default, the source files are not included.
+Hidden files (starting with a dot) are not included in the bundle.
+
+#### --format
+
+The format of the output bundle. Supported formats are `zip` and `tgz`. Default is `tgz`.
+
+#### --prefix
+
+The directory where the output bundle will be saved. Default is `.`.
```
Comment (yury-palyanitsa) [inline README.md:107]: Why both `--prefix` and `--output`? Isn't just `--output` already enough?

Comment (IKukhta) [reply README.md:128]: just a standard approach

**Review by viatoriche — APPROVED** [review]

**Review by yury-palyanitsa — APPROVED** [review]

**Pull request merged**